docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  postgres:
+    image: postgres:15-alpine
+    environment:
+      - POSTGRES_PASSWORD=postgres
+    volumes:
+      - ./docker/init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
+    ports:
+      - 5432:5432
```

Comment:
Yep just apply your init to the woodpecker.yml .

Reply:
Done, and added to `docker/docker-compose.yml` too so the CI and local behavior is the same.